netbox/utilities/templatetags/form_helpers.py

```diff
@@ -32,6 +34,14 @@ def getfield(form, fieldname):
         return None
 
 
+@register.filter()
+def get_filter_field(form, fieldname):
+    # Check for a table form column map attribute and use that to map form fields if set
+    if hasattr(form, '_table_form_column_map') and form._table_form_column_map.get(fieldname):
+        return getfield(form, form._table_form_column_map.get(fieldname))
```

Comment:
Not set on this, but also wondering if this should be a default on NetBoxTable

Reply:
Where is `_table_form_column_map` getting set? If it's not yet needed, let's ditch it for now.